components/PriortyBillsCard/PriorityBillsCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { Card } from "react-bootstrap"
+import Styles from "./PriorityBillsCard.module.css"
+
+type bill = {
+  id: string
+  billNumber: string
+  title: string
+  approval: string
+}
+
+export const PriorityBillsCard = (props: {
+  bills: bill[]
+  bill_id: string
+  session: string
+  onClick: any
+}) => {
+  return (
+    <>
+      <Card className={Styles.header}>
```

Comment:
We should try to use the generic card component that David merged in last week: https://master--634f3926f2a0d0f0195eefd7.chromatic.com/?path=/story/components-cards-card--header-and-subheader-and-body-and-two-list-items I think you'll need to pull the latest code for it to show up in your branch.